packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -108,7 +122,7 @@ export class ColorPicker extends React.Component<IColorPickerProps, any> {
     if (newColor.str !== this.state.color.str) {
       this.setState({
         color: newColor
-      });
+      } as IColorPickerState);
 
       if (onColorChanged) {
         onColorChanged(newColor.str);
```

Comment:
This should, I think, really be a callback with the setState. Otherwise it's asynchronous and could casue problems. So it should be something like
``` 
this.setState({
    color: newColor
}, () => { 
    if (onColorChanged) { 
        onColorChanged(newColor.str); 
    }
}
```